src/devTools.js

```diff
@@ -95,13 +107,31 @@ class DevToolsEnhancer {
     }
   }
 
+  getLiftedState = (store, filters) => {
+    const fixedStore = store || this.store;
```

Comment:
this comes from the fact that I overwrote an existing `getLiftedState` function(the old one wouldn't receive any args). This code will be fixed.